README.md

```diff
@@ -1,291 +1,234 @@
-- Contributing
-- Show your support
-- Acknowledgements
-- License
-
-OPTIONAL SECTIONS:
-- FAQ
-
-After you're finished please remove all the comments and instructions!
-
-For more information on the importance of a professional README for your repositories: https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/readme_best_practices.md
--->
-
 <div align="center">
-  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
-  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
   <br/>
 
-  <h3><b>Microverse README Template</b></h3>
+  <h1><b>Metrics</b></h1>
 
 </div>
 
-<!-- TABLE OF CONTENTS -->
-
 # 📗 Table of Contents
 
-- [📖 About the Project](#about-project)
-  - [🛠 Built With](#built-with)
-    - [Tech Stack](#tech-stack)
-    - [Key Features](#key-features)
-  - [🚀 Live Demo](#live-demo)
-- [💻 Getting Started](#getting-started)
-  - [Prerequisites](#prerequisites)
-  - [Setup](#setup)
-  - [Install](#install)
-  - [Usage](#usage)
-  - [Run tests](#run-tests)
-  - [Deployment](#deployment)
-- [👥 Authors](#authors)
-- [🔭 Future Features](#future-features)
-- [🤝 Contributing](#contributing)
-- [⭐️ Show your support](#support)
-- [🙏 Acknowledgements](#acknowledgements)
-- [❓ FAQ (OPTIONAL)](#faq)
-- [📝 License](#license)
-
-<!-- PROJECT DESCRIPTION -->
-
-# 📖 [your_project_name] <a name="about-project"></a>
-
-> Describe your project in 1 or 2 sentences.
-
-**[your_project__name]** is a...
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Metrics Tracker ](#-metrics-tracker-)
+ - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Project Structure](#project-structure)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+# 📖 Metrics Tracker <a name="about-project"></a>
+
+>Welcome to Metricks Tracker, a mobile web app using React and Redux! Explore a selected API, monitor, and obtain information effortlessly! Metricks Tracker focuses primarily on the world population to offer a detailed and meaningful analysis of data related to this area. With Metricks Tracker, you will have access to valuable and up-to-date information about the world population, enabling you to make informed decisions and gain a clearer understanding of demographic patterns and trends. Start using Metricks Tracker now and discover a world of statistical information at your fingertips!.
+[Here](https://www.loom.com/share/148a4deb8986456e99c918d9a9387194c?sid=837942a4-36f1-4d20-aac3-a9464faixq2c0b) is the link to the video presentation 
```

Comment:
> * [x]  **Good job so far 👏 👏** However, it would be better to provide a valid link to the Presentation video in your README as suggested in the project requirements.
>   Currently, the provided link gives the following:
> 
> <img alt="Screen Shot 2023-10-13 at 18 02 59" width="1440" src="https://user-images.githubusercontent.com/102757126/274954029-33c87e24-e92d-4ec8-b894-fb6d958c64e3.png">
> Please remember, you should record a video presenting the features of the project you built. It is a video with a maximum length of 5 minutes. The content of the video should include:
> 
> ```
> - a description of the project.
> - a demo of the project features.
> - you should also highlight some interesting piece of code or something you built that you are very proud of.
> ```
> 
> For recording the video you can use tools like [Loom](https://www.loom.com/) that let you share a private link to the recording, and configure a shot that shows your computer screen and your face at the same time in a small picture.


- [ ] Hello, as mentioned by the previous reviewer, Kindly have the link to the live video presentation fix, I get below error while accessing the video.

![Screenshot from 2023-10-14 23-09-04](https://github.com/gdvl777/metrics/assets/112261237/a246cc51-0b27-4efe-aa0b-aad7b23b8d1a)